wait/host_port_test.go

```diff
@@ -508,3 +508,58 @@ func TestHostPortStrategySucceedsGivenShellIsNotInstalled(t *testing.T) {
 	err = wg.WaitUntilReady(context.Background(), target)
 	require.NoError(t, err)
 }
+
+func TestHostPortStrategySucceedsGivenShellIsNotFound(t *testing.T) {
+	listener, err := net.Listen("tcp", "localhost:0")
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
suggestion: could we use require for all checks please, just makes the tests much cleaner, for example:
```suggestion
	require.NoError(t, err)
```

Reply:
done. `require` sounds much better for sure.